Header header
uint32 height
uint32 width
float32 tau_el
float32 tau_er
float32 tau_l
float32 tau_r
float32 tau_c
